config/webpack.test.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path')
+const VueLoaderPlugin = require('vue-loader/lib/plugin')
+
+const config = {
```

Comment:
So there's no difference between the .test and .development config files?

Reply:
not right now I would say